docs/helm.md

```diff
@@ -0,0 +1,129 @@
+# Installing Antrea with Helm
+
+## Table of Contents
+
+<!-- toc -->
+- [Prerequisites](#prerequisites)
+- [Charts](#charts)
+  - [Antrea chart](#antrea-chart)
+    - [Installation](#installation)
+    - [Upgrade](#upgrade)
+    - [An important note on CRDs](#an-important-note-on-crds)
+  - [Flow Aggregator chart](#flow-aggregator-chart)
+    - [Installation](#installation-1)
+    - [Upgrade](#upgrade-1)
+  - [Theia chart](#theia-chart)
+<!-- /toc -->
+
+Starting with Antrea v1.8, Antrea can be installed and updated using
+[Helm](https://helm.sh/).
+
+We provide the following Helm charts:
+
+* `antrea/antrea`: the Antrea network plugin
+* `antrea/flow-aggregator`: the Antrea Flow Aggregator; see
+  [here](network-flow-visibility.md) for more details.
+* `antrea/theia`: Theia, the Antrea network observability solution; refer to the
+  [Theia](https://github.com/antrea-io/theia) sub-project for more details.
+
+Note that these charts are the same charts that we use to generate the YAML
+manifests for the `kubectl apply` installation method.
+
+Helm installation is currently considered Alpha.
+
+## Prerequisites
+
+* Ensure that the necessary
+  [requirements](getting-started.md#ensuring-requirements-are-satisfied) for
+  running Antrea are met.
+* Ensure that Helm 3 is [installed](https://helm.sh/docs/intro/install/). We
+  recommend using a recent version of Helm if possible. Refer to the [Helm
+  documentation](https://helm.sh/docs/topics/version_skew/) for compatibility
+  between Helm and Kubernetes versions.
+* Add the Antrea Helm chart repository:
+
+  ```bash
+  helm repo add https://charts.antrea.io
+  helm repo update
+  ```
+
+## Charts
+
+### Antrea chart
+
+#### Installation
+
+To install the Antrea Helm chart, use the following command:
+
+```bash
+helm install antrea antrea/antrea --namespace kube-system
+```
+
+This will install the latest available version of Antrea. You can also install a
+specific version of Antrea (>= v1.8.0) with `--version <TAG>`.
```

Comment:
Just to check if the latest version will be generated automatically? Based on my test, running `helm install antrea antrea/antrea --namespace kube-system` without version will get an error saying `Error: INSTALLATION FAILED: failed to download "antrea/antrea"`. Not sure if I get something wrong.

Reply:
This is expected. You have to use the `--devel` flag at the moment, since the only chart releases we have currently are Alpha releases. When `v1.8.0` is released, this command will work fine.

```
$ helm search repo --devel --regexp antrea
NAME                  	CHART VERSION	APP VERSION  	DESCRIPTION
antrea/antrea         	1.8.0-alpha.2	1.8.0-alpha.2	Kubernetes networking based on Open vSwitch
antrea/flow-aggregator	1.8.0-alpha.2	1.8.0-alpha.2	Antrea Flow Aggregator
antrea/theia          	0.2.0-alpha.0	0.2.0-alpha.0	Antrea Network Flow Visibility
```

Reply:
Got it. Thanks for the explanation!